doc/design/upgrade.md

```diff
@@ -26,7 +45,9 @@ There will be impact on container interaction while the appliance is down:
 - no attach ability
 - network outage for NAT based port forwarding
 
-All of those facets should resume normal operation after upgrade is complete
+All of those facets should resume normal operation after upgrade is complete. 
```

Comment:
@hmahmood doesn't impact @emlin but which field in the contianer config? I can see exposed ports, but nothing which port they're forwarded to.